arelle/CntlrCmdLine.py

```diff
@@ -556,10 +557,19 @@ def filesourceEntrypointFiles(filesource, entrypointFiles=[], inlineOnly=False):
 
     elif os.path.isdir(filesource.url):
         del entrypointFiles[:] # clear list
+        hasInline = False
         for _file in os.listdir(filesource.url):
             _path = os.path.join(filesource.url, _file)
-            if os.path.isfile(_path) and ModelDocument.Type.identify(filesource, _path) in (ModelDocument.Type.INSTANCE, ModelDocument.Type.INLINEXBRL):
-                entrypointFiles.append({"file":_path})
+            if os.path.isfile(_path):
+                identifiedType = ModelDocument.Type.identify(filesource, _path)
+                if identifiedType == ModelDocument.Type.INLINEXBRL:
+                    hasInline = True
+                if identifiedType in (ModelDocument.Type.INSTANCE, ModelDocument.Type.INLINEXBRL):
+                    entrypointFiles.append({"file":_path})
+        if hasInline: # group into IXDS if plugin feature is available
+            for pluginXbrlMethod in pluginClassMethods("InlineDocumentSet.Discovery"):
```

Comment:
Does this still work without the ixds plugin for zip files with a single inline doc?